ops/framework.py

```diff
@@ -852,33 +852,34 @@ def __get__(self, parent, parent_type=None):
             # accessing via the class directly (e.g. MyClass.stored)
             return self
 
-        if self.attr_name is not None and self.attr_name in parent.__dict__:
-            # we probably already have the thing
-            bound = parent.__dict__[self.attr_name]
-
-            if bound is not None:
-                return bound
+        bound = parent.__dict__.get(self.attr_name)
```

Comment:
The attr_name is None check here still feels proper. It informs the reader that this is a possibility and prevents attempting to get None from the __dict__.